.github/setup.yml

```diff
@@ -0,0 +1,91 @@
+name: "Perform Initial Repository Setup"
```

Comment:
Have updated this workflow file essentially it is designed to be run once then deletes it self, just sets up tooling around the draft publication